tests/textlab_api/textlab_api_test.py

```diff
@@ -13,3 +32,316 @@ def test_normalize_text(
     Test for the text normalization function that is applied before sending text to the TextLab API
     """
     assert output == normalize_text(input)
+
+
+def update_page_content(
+    login_role_user: tuple[Client, str],
+    title: str,
+    content: str,
+    hix_ignore: bool = False,
+) -> tuple[str, HttpResponse]:
+
+    # Log the user in
+    client, _role = login_role_user
+
+    # Update page content
```

Comment:
```suggestion
```
Nit-picky comment:
Since the whole function is called "update_page_content", I think the additional clarifying comment can be omitted.